src/hooks/useActions.ts

```diff
@@ -29,7 +30,9 @@ const SIGN_GAS_LIMIT = 100000
 const STAKE_GAS_LIMIT = 250000
 const WRAP_GAS_LIMIT = 1000000
 
-export default function useActions() {
+export default function useActions(): {
+  [x: string]: any
```

Comment:
why is this needed since is not receiving any parm?

Reply:
Its typed return of function, note its after `():`

Its type its kinda generic object, dynamic properties string based, the better way should be, what its exactly returns.

Reply:
Looks like already exist some type we can use on return that, its right?

https://github.com/1Hive/gardens-ui/pull/771/files?diff=split&w=0#diff-04581b471ddd0404521d07d207d7ceb2529e3c4d9d73550d8bf8698ed09f51b9R234

Reply:
I created that ActionsType but forgot to add to useActions. Already fixed.

Reply:
![Uploading Captura de ecrã 2022-01-05, às 10.12.48.png…]()